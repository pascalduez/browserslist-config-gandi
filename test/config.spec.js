const browserslist = require('browserslist');
const config = require('../');

const browsers = browserslist(config);

describe('browserslist-config-gandi', () => {
  it('should successfully resolves a list of browsers', () => {
    expect(browsers.length).toBeGreaterThan(0);
    expect(browsers).toMatchSnapshot();
  });

  afterAll(() => {
    console.log('Resolved the following browsers:\n\n', browsers.join('\n'));
  });
});
