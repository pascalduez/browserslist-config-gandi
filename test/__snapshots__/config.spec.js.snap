// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browserslist-config-gandi should successfully resolves a list of browsers 1`] = `
Array [
  "and_chr 61",
  "and_ff 56",
  "and_qq 1.2",
  "and_uc 11.4",
  "android 56",
  "android 4.4.3-4.4.4",
  "baidu 7.12",
  "bb 10",
  "bb 7",
  "chrome 61",
  "chrome 60",
  "chrome 49",
  "edge 15",
  "edge 14",
  "edge 13",
  "firefox 56",
  "firefox 55",
  "ie 11",
  "ie 10",
  "ie_mob 11",
  "ie_mob 10",
  "ios_saf 11",
  "ios_saf 10.3",
  "ios_saf 10.0-10.2",
  "ios_saf 9.3",
  "ios_saf 9.0-9.2",
  "ios_saf 8.1-8.4",
  "ios_saf 8",
  "ios_saf 7.0-7.1",
  "ios_saf 6.0-6.1",
  "op_mini all",
  "op_mob 37",
  "op_mob 12.1",
  "opera 48",
  "opera 47",
  "safari 11",
  "safari 10.1",
  "safari 10",
  "safari 9.1",
  "safari 9",
  "safari 8",
  "safari 7.1",
  "safari 7",
  "safari 6.1",
  "safari 6",
  "samsung 5",
  "samsung 4",
]
`;
